### ISSUE TITLE
MakeSearchable runs twice when updating aggregated model

### ISSUE THREAD

**Issue by aniplaylist** [algolia/scout-extended#237]
Hello, 

I have just set up an Aggregator on my project with two Models in it.

Whenever I'm updating one of these two models the MakeSearchable Job is run twice : 
- Once for the Model
- Once for the Aggregator

Is this the expected behaviour? Isn't this consuming two indexing operations on Algolia instead of one?

**Two jobs seen on Horizon :** 

<img width="922" alt="horizon-jobs" src="https://user-images.githubusercontent.com/40739069/82935987-2bf6fa00-9f8e-11ea-9a08-ebca9a29b34e.png">

**Model :** 

<img width="832" alt="jobs-103-model" src="https://user-images.githubusercontent.com/40739069/82935991-2d282700-9f8e-11ea-8bb5-d1ead1e752dc.png">

**Aggregator :**
 
<img width="927" alt="jobs-104-aggregator" src="https://user-images.githubusercontent.com/40739069/82935997-2ef1ea80-9f8e-11ea-8438-2e0460b36076.png">

**Comment by nunomaduro** [comment]
Correct. If both the model, and the aggregator are searchable that is the normal behaviour.

You really need to make the models searchable? If not, you can remove the `Searchable` trait from those models, and the aggregator will still work consuming just one indexing operation per update.

**Comment by aniplaylist** [comment]
Just tried removing the `Searchable` trait from my model but `MakeSearchable` job is not triggered anymore (not even once) and the entry I updated is removed from my index

**Comment by nunomaduro** [comment]
That is because you probably forgot to call the `bootSearchable` in your service provider as specified here: [algolia.com/doc/framework-integration/laravel/advanced-use-cases/multiple-models-in-one-index/?language=php#aggregators---multiple-models-in-one-index](https://www.algolia.com/doc/framework-integration/laravel/advanced-use-cases/multiple-models-in-one-index/?language=php#aggregators---multiple-models-in-one-index).

**Comment by aniplaylist** [comment]
I didn't forgot 🤔 

```
<?php

namespace App\Providers;

use App\Search\Results;
use Illuminate\Support\ServiceProvider;

class AppServiceProvider extends ServiceProvider
{
    /**
     * Bootstrap any application services.
     *
     * @return void
     */
    public function boot()
    {
        Results::bootSearchable();
    }

    /**
     * Register any application services.
     *
     * @return void
     */
    public function register()
    {
    }
}
```

**Comment by tech-commoprices** [comment]
Hi @aniplaylist , just ran into the same issue.

Simply change the `shouldBeSearchable` method on your Aggregator to make it look like this :

```
    public function shouldBeSearchable()
    {
        return $this->model->shouldBeSearchable();
    }
```

**Comment by tech-commoprices** [comment]
It could be useful to put this in the docs btw :-)

**Comment by nunomaduro** [comment]
Thanks! We are aware of this issue, no ETA for having this on the package at the moment.
